babel.config.js

```diff
@@ -46,9 +27,25 @@ const productionPlugins = [
   ],
 ];
 
-module.exports = {
-  presets: defaultPresets.concat(['@babel/preset-react', '@babel/preset-typescript']),
-  plugins: [
+module.exports = function getBabelConfig(api) {
+  const useESModules = api.env(['legacy', 'modern', 'stable']);
```

Comment:
Am I reading this right that legacy SHOULD use ESModules? This is almost certainly a mistake, right?

Reply:
I believe this stands for _legacy ESM_.

From [microbundle readme](https://github.com/developit/microbundle):
```json
	"module": "./dist/foo.module.js", // legacy ES Modules output (for bundlers)
	"exports": "./dist/foo.modern.js", // modern ES2017 output
```

Reply:
> Am I reading this right that legacy SHOULD use ESModules? This is almost certainly a mistake, right?

Could you open an issue and fill out the template to describe the mistake? Discussions on merged PRs are oftentimes forgotten.

Reply:
I guess I thought legacy was for ancient browsers like ie11, it seems that I might be incorrect about that. Thanks for the explanation, sorry for the mis-informed question. Have a nice day.

Reply:
legacy is for IE11. But IE11 does not understand ES modules nor CommonJS modules i.e. the code always needs a bundler anyway at which point we might as well ship ES modules to help tree-shaking.